readyset-example/app.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+
+import argparse
+import psycopg2
+import time
+import numpy as np
+
+parser = argparse.ArgumentParser(
+    description="test performance of ReadySet vs. a backing Postgres database")
+parser.add_argument("--query",
+                    required=False,
+                    help="query to execute")
+parser.add_argument("--repeat",
+                    type=int,
+                    help="number of times to run the query",
+                    default = 20)
+parser.add_argument("--url",
+                    required=True,
+                    help="connection URL for ReadySet or Postgres")
+args = parser.parse_args()
+
+query = "SELECT count(*) FROM title_ratings JOIN title_basics ON title_ratings.tconst = title_basics.tconst WHERE title_basics.startyear = 2000 AND title_ratings.averagerating > 5"
+
+print(args.url)
+conn = psycopg2.connect(dsn=args.url)
+conn.set_session(autocommit=True)
+cur = conn.cursor()
+
+times = list()
+for n in range(args.repeat):
+    start = time.time()
+    cur.execute(query)
+    if n < 1:
```

Comment:
why you print the result on the first iteration only?

Reply:
it's same value for all the iterations